### PR TITLE
Pbr from pdf

### PR DESCRIPTION
**Pull request by jbrou015**

### CODE REVIEW

**Comment by joshcampbell191** [comment]
@jbrou015 Can you please add jqueryui to the libman.json and ensure the folder is ignored using the .gitignore. This way, we don't need to include all the resources in our project?

File: ComplaintFormCore/wwwroot/sample-data/survey_pbr_2.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
```
Comment (joshcampbell191) [inline ComplaintFormCore/wwwroot/sample-data/survey_pbr_2.json:1]: I assume we don't need this file as it was created for testing purposes?

Comment (jbrou015) [reply ComplaintFormCore/wwwroot/sample-data/survey_pbr_2.json:1]: That's right

File: ComplaintFormCore/Views/Home/Pbr.cshtml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+@section CustomStyles {
+    <link href="~/css/site.css" rel="stylesheet" />
+    <link href="~/css/ladda-themeless.min.css" rel="stylesheet" />
+    <link rel="stylesheet" href="~/lib/jqueryui/jquery-ui.min.css" />
+}
+
+@section CustomScripts {
+    <script src="~/lib/jqueryui/jquery-ui.min.js"></script>
+    <script src="~/dist/bundle.js"></script>
+    <script src="~/lib/surveyjs-widgets/widgets/jquery-ui-datepicker.js"></script>
```
Comment (joshcampbell191) [inline ComplaintFormCore/Views/Home/Pbr.cshtml:42]: I don't think we need this as it's bundled with the ClientApp. I didn't notice that the first time around.

File: ComplaintFormCore/libman.json [hunk 3]
```diff
@@ -2,5 +2,14 @@
...
   "libraries": [
+    {
+      "library": "jqueryui@1.12.1",
+      "destination": "wwwroot/lib/jqueryui/"
+    },
+    {
+      "provider": "jsdelivr",
+      "library": "surveyjs-widgets@1.8.13",
+      "destination": "wwwroot/lib/surveyjs-widgets/"
+    }
```
Comment (joshcampbell191) [inline ComplaintFormCore/libman.json:13]: I don't think this is required anymore as it's bundled with the ClientApp. Can you please test to ensure this?

**Review by joshcampbell191 — APPROVED** [review]

**Pull request merged**